### PR TITLE
Clean the rep + swiftLint

### PR DESCRIPTION
**Pull request by AntoineJac**
I have used Swiftlint to clean the repo. 
I have also added a swiftlint.yml rules for the PrebidMobile directory and edit the travis.
Anycode not compliant with the swiftlint.yml rules will now fail.

### CODE REVIEW

**Comment by AntoineJac** [comment]
@ppuviarasu  and @yoalex5 , here is a PR with:
- clean of the repo
- add swiftLint to current Travis
- add the InventoryKey to request (userInventory is already present)

Thanks,

File: src/PrebidMobile/PrebidMobile/AdUnits/AdUnit.swift [hunk 1]
```diff
@@ -16,184 +16,233 @@
...
-        Log.info("user keywords are \(customKeywords)")
-        return customKeywords
+
+    var userKeywords: [String: [String]] {
+        Log.info("user keywords are \(customUserKeywords)")
+        return customUserKeywords
     }
-    
+
+    var invKeywords: [String: [String]] {
```
Comment (ppuviarasu) [inline src/PrebidMobile/PrebidMobile/AdUnits/AdUnit.swift]: This is a new API addition... Were is the requirements for this? We need to have the feature created before coding in. Please create a feature request first and have the products approval before adding in the code for the same

Comment (AntoineJac) [reply src/PrebidMobile/PrebidMobile/AdUnits/AdUnit.swift]: @ppuviarasu , here is the feature request: #165

Comment (ppuviarasu) [reply src/PrebidMobile/PrebidMobile/AdUnits/AdUnit.swift]: As @andrew suggested, please split the PR. Keep the feature PR separate

Comment (AntoineJac) [reply src/PrebidMobile/PrebidMobile/AdUnits/AdUnit.swift]: Hi @ppuviarasu , sure. I have removed it and add it in a new PR:
https://github.com/prebid/prebid-mobile-ios/pull/166

Thanks

**Comment by ppuviarasu** [comment]
Can you resolve the conflicts and push in?

**Comment by AntoineJac** [comment]
@ppuviarasu , conflicts are solved. Can you please merge?

Thanks,

**Comment by AntoineJac** [comment]
@ppuviarasu , can you please merge when ready ?

**Comment by ppuviarasu** [comment]
checks are not passing... something is not rite... looking into it

**Comment by AntoineJac** [comment]
> checks are not passing... something is not rite... looking into it
Hi @ppuviarasu ,
Any update on this?
It is maybe because I am updating the current travis file.
Thanks,

**Comment by ppuviarasu** [comment]
No... it doesnt work... I dont see the changes different in the travis file. Can you revert the same and push?
Only if the checks pass it will allow me to merge the same

**Comment by AntoineJac** [comment]
@ppuviarasu , can you please merge. It is now working. Thanks.

**Pull request merged**